poglink/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .bans import BansPlatformPair, BansStatus, BansTimePeriodSummary
+from .rates import RatesDiff, RatesDiffItem, RatesStatus
```

Comment:
I've imported these here only for compatibility, so the import is still `from poglink.models import RatesDiff` or whatever.

A more explicit form would now be `from poglink.models.rates import RatesDiff`, but either will work as of this change.